movieapp/src/containers/SearchForm/SearchForm.tsx

```diff
@@ -1,11 +1,13 @@
 import { Button } from '../../components/Button/Button';
 import { SearchField } from '../../components/SearchField/SearchField';
+import { useTranslation } from "react-i18next";
 
 export function SearchForm(): JSX.Element {
+    const { t } = useTranslation();
```

Comment:
indents should be handled by eslint as well.

add husky pre-push hook to run eslint

Reply:
fixed